src/AltinnCore/ServiceLibrary/Api/ApiValidationResult.cs

```diff
@@ -16,5 +16,10 @@ public class ApiValidationResult
         /// Gets or sets the collection of warnings
         /// </summary>
         public Dictionary<string, List<string>> Warnings { get; set; }
```

Comment:
Guessing Warnings and Errors are no longer used? If so these can be removed

Reply:
Not for api response to the client, but might still be relevant for EUS. They are used in the other method mapping api response, used in the corresponding POST method